### ISSUE TITLE
Adding a TapGestureRocognizer not working on Android

### ISSUE THREAD

**Issue by craylward** [matteobortolazzo/HtmlLabelPlugin#35]
### Description
Adding a TapGestureRecognizer to HtmlLabel isn't picking up taps on Android only. iOS works fine.

Works fine with Label instead of HtmlLabel.

### Version
Xamarin.Forms 3.0.0.561731
Xam.Plugin.HtmlLabel 3.0.1.3-beta

```
<htmlLabel:HtmlLabel
	VerticalOptions="CenterAndExpand"
	HorizontalTextAlignment="Center"
	Margin="50,0,50,0"
	Text="Test">
	<Label.GestureRecognizers>
		<TapGestureRecognizer Command="{Binding Clicked}" />
	</Label.GestureRecognizers>
</htmlLabel:HtmlLabel>
```

**Comment by seddik** [comment]
as mentioned in #29 its still doesn't work on android even when using `Label.GestureRecognizers`, my hacky solution is to put it in a grid the put another grid as the last child, and bind the gesture to the grid, but its not a good solution

**Comment by MohanaSundharamMurugan** [comment]
you can use like this,
<StackLayout Spacing=0>
<StackLayout.GestureRecognizers>
		<TapGestureRecognizer Command="{Binding Clicked}" />
	</StackLayout.GestureRecognizers>
<htmlLabel:HtmlLabel
	VerticalOptions="CenterAndExpand"
	HorizontalTextAlignment="Center"
	Margin="50,0,50,0"
	Text="Test">
</htmlLabel:HtmlLabel>
</StackLayout>

**Comment by matteobortolazzo** [comment]
Fixed in [https://github.com/matteobortolazzo/HtmlLabelPlugin/commit/08c20cc6a4decfd19c39769f2a9fa2ec4235b9db](https://github.com/matteobortolazzo/HtmlLabelPlugin/commit/08c20cc6a4decfd19c39769f2a9fa2ec4235b9db) (But it disables links)

**Comment by matteobortolazzo** [comment]
Closed with #80
